Pod/Classes/TEDomain.m

```diff
@@ -94,6 +94,16 @@ - (void)sendActionToMiddlewares:(TEBaseAction *)action
     }];
 }
 
+- (void)registerTemporaryStore:(TEBaseStore *)store {
```

Comment:
you can have registerStore method and registerTemporaryStore and registerGlobalStore